website/docs/Project-API.md

```diff
@@ -72,6 +72,7 @@
 | [editandcontinue](editandcontinue.md)                     |  |
 | [editorintegration](editorintegration.md)                 | Enable or disable IDE integration |
 | [enablewarnings](enablewarnings.md)                       |  |
+| [enableunitybuilds](unitybuilds.md)                       |  |
```

Comment:
Our current convention in docs is that file names are the same as the function that they describe. You have added the `enableUnityBuilds` API, so the file should be named `enableUnityBuilds.md`. Lua is case-sensitive, so the above line is also misleading 😉 Please change it to:

```markdown
| [enableUnityBuilds](enableUnityBuilds.md)                       |  |
```

Reply:
I fixed up all the naming to match what is already there, I hope this is more consistent now.